--- cobbler/action_check.py	2013-11-11 13:56:35.926916887 +0100
+++ cobbler/action_check.py	2013-11-11 19:03:23.237715702 +0100
@@ -123,10 +123,10 @@
            if rc != 0:
                status.append(_("service %s is not running%s") % (which,notes))
                return False
-       elif self.checked_dist in ["debian", "ubuntu"]:
+       elif self.checked_dist in ["debian", "ubuntu"] or self.checked_dist.find("gentoo") > -1:
            # we still use /etc/init.d
            if os.path.exists("/etc/init.d/%s" % which):
-               rc = utils.subprocess_call(self.logger,"/etc/init.d/%s status /dev/null 2>/dev/null" % which, shell=True)
+               rc = utils.subprocess_call(self.logger,"/etc/init.d/%s status > /dev/null 2>/dev/null" % which, shell=True)
            if rc != 0:
                status.append(_("service %s is not running%s") % (which,notes))
                return False
@@ -408,14 +408,19 @@
        if self.checked_dist in ["debian", "ubuntu"]:
           return
 
-       if os.path.exists("/etc/xinetd.d/rsync"):
-          f = open("/etc/xinetd.d/rsync")
+       rsync_conf_path = "/etc/xinetd.d/rsync"
+
+       if self.checked_dist.find("gentoo") > -1:
+          rsync_conf_path = "/etc/xinetd.d/rsyncd"
+
+       if os.path.exists(rsync_conf_path):
+          f = open(rsync_conf_path)
           re_disable = re.compile(r'disable.*=.*yes')
           for line in f.readlines():
              if re_disable.search(line) and not line.strip().startswith("#"):
-                 status.append(_("change 'disable' to 'no' in %(file)s") % { "file" : "/etc/xinetd.d/rsync" })
+                 status.append(_("change 'disable' to 'no' in %(file)s") % { "file" : rsync_conf_path })
        else:
-          status.append(_("file %(file)s does not exist") % { "file" : "/etc/xinetd.d/rsync" })
+          status.append(_("file %(file)s does not exist") % { "file" : rsync_conf_path })
 
 
    def check_dhcpd_conf(self,status):
